pages/1.11/deploying-services/marathon-on-marathon/advanced/index.md

```diff
@@ -10,70 +10,95 @@ enterprise: true
 
 This topic describes how to deploy a non-native instance of Marathon with isolated roles, reservations, quotas, and security features. The advanced non-native Marathon procedure should only be used if you require [secrets](/1.11/security/ent/secrets/) or fine-grain ACLs, otherwise use the [basic procedure](/1.11/deploying-services/marathon-on-marathon/basic/).
 
-To use this procedure, you must obtain the custom non-native Marathon tarball from your sales representative (<sales@mesosphere.io>). This custom tarball contains Marathon plus a secrets and auth plugin. These additional plugins allow you to use secrets and fine-grained access control in your non-native Marathon folder hierarchy.
+For this procedure, we are assuming that you have obtained an enterprise version of Marathon from your sales representative (<sales@mesosphere.io>). This custom tarball contains Marathon plus secrets and auth plugins. These additional plugins allow you to use secrets and fine-grained access control in your non-native Marathon folder hierarchy.
```

Comment:
`auth plugins` is "authentication and authorization plugins"? I would prefer it being entirely deterministic if so.

Reply:
Can you clarify what you're asking here, tillt?